code.gs

```diff
@@ -561,12 +561,13 @@ Contact.prototype.getLines = function (type, date, format) {
     }
     // Custom label
     if (type === 'CUSTOM') {
+      var customEventLabel = event.getProp('label') || 'OTHER';
```

Comment:
I'd move the variable declaration to line R359 to stick with the coding style (or get rid of the variable altogether and place `|| 'OTHER'` twice inline, but I prefer using the variable since it would be useful if more notification types are added in the future).